test/test_mkldnn.py

```diff
@@ -1037,6 +1037,66 @@ def test_resnext50_32x4d(self):
         model = torchvision.models.resnet.resnext50_32x4d(pretrained=False)
         self._test_imagenet_model(model)
 
+    @unittest.skipIf(IS_WINDOWS, "Limit support for bf16 path")
```

Comment:
SO....
- try to benchmark nn.Linear and make sure we don't have redundant memory copy.
- add test from xxx.out ( i suppose this should already be covered in test_nn.py).
- i am not sure whether memory overlapping cases are needed